csharp/ql/test/library-tests/dataflow/library/FlowSummaries.expected

```diff
@@ -3285,41 +3323,6 @@ summary
 | Microsoft.VisualBasic;VBCodeProvider;VBCodeProvider;(System.Collections.Generic.IDictionary<System.String,System.String>);Argument[0].Element;Argument[this];taint;df-generated |
 | Microsoft.Win32.SafeHandles;SafeFileHandle;SafeFileHandle;(System.IntPtr,System.Boolean);Argument[0];Argument[this].Field[System.Runtime.InteropServices.SafeHandle.handle];value;dfc-generated |
 | Microsoft.Win32.SafeHandles;SafeWaitHandle;SafeWaitHandle;(System.IntPtr,System.Boolean);Argument[0];Argument[this].Field[System.Runtime.InteropServices.SafeHandle.handle];value;dfc-generated |
-| Microsoft.Win32;PowerModeChangedEventHandler;BeginInvoke;(System.Object,Microsoft.Win32.PowerModeChangedEventArgs,System.AsyncCallback,System.Object);Argument[2];Argument[2].Parameter[delegate-self];value;hq-generated |
```

Comment:
Is it expected that we loose so many rows?

Reply:
This is because some of dependencies have changed such that `Microsoft.Windows.Extensions` is no longer referenced in the updated versions of the packages. If this is of concern, I can take a look and see whether se should try to explicitly include these packages.